lib/checks/aria/aria-required-children-evaluate.js

```diff
@@ -15,6 +20,10 @@ function getOwnedRoles(virtualNode, required) {
   const ownedElements = getOwnedVirtual(virtualNode);
   for (let i = 0; i < ownedElements.length; i++) {
     const ownedElement = ownedElements[i];
+    if (ownedElement.props.nodeType !== 1) {
```

Comment:
Needed to ignore non-element nodes as `isVisibletoScreenreaders` calls into `isHiddenForEveryone` which tries to get the computed style for `display`, which fails for non-element nodes